src/ValueParsers/CalendarModelParser.php

```diff
@@ -30,11 +30,12 @@ protected function stringParse( $value ) {
 		switch ( $value ) {
 			case '':
 			case 'gregorian':
-				return TimeFormatter::CALENDAR_GREGORIAN;
+				return IsoTimestampParser::CALENDAR_GREGORIAN;
```

Comment:
Having the constants in `IsoTimestampParser` seems rather arbitrary to me as well. Just saying, not objecting to this change.